.github/workflows/release.yaml

```diff
@@ -0,0 +1,75 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v*
+  workflow_dispatch:
+    inputs:
+      latest:
+        required: false
+        type: boolean
+        default: false
+      tag:
+        required: true
+        type: string
+        description: 'Release tag (chart and images)'
+        
+jobs:
+  set-variables:
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.set-vars.outputs.tag }}
+      latest: ${{ steps.set-vars.outputs.latest }}
+    steps:
+      - id: set-vars
+        shell: bash
+        env:
+          EVENT_NAME: ${{ github.event_name }}
+          REF: ${{ github.ref }}
+          INPUT_TAG: ${{ github.event.inputs.tag }}
+          INPUT_LATEST: ${{ github.event.inputs.latest }}
+        run: |
+          if [[ "$EVENT_NAME" == "push" && "$REF" == refs/tags/v* ]]; then
+            echo "Tag push event"
+            TAG="${REF##*/}"
+            LATEST="true"
+          else
+            echo "Workflow dispatch or other event"
+            TAG="$INPUT_TAG"
+            # Ensure INPUT_LATEST is 'true' or 'false' string
+            if [[ "$INPUT_LATEST" == "true" ]]; then
+              LATEST="true"
+            else
+              LATEST="false"
+            fi
+          fi
+          echo "tag=$TAG" >> $GITHUB_OUTPUT
+          echo "latest=$LATEST" >> $GITHUB_OUTPUT
+
+
+  release-images:
+    needs: set-variables
+    uses: ./.github/workflows/release-images.yaml
+    with:
+      tag: ${{ needs.set-variables.outputs.tag }}
+      latest: '${{ fromJSON(needs.set-variables.outputs.latest) }}'
+    secrets:
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+
+  release-charts:
+    needs: [set-variables, release-images]
+    uses: ./.github/workflows/release-charts.yaml
+    with:
+      chart_tag: ${{ needs.set-variables.outputs.tag }}
+      images_tag: ${{ needs.set-variables.outputs.tag }}
+      latest: '${{ fromJSON(needs.set-variables.outputs.latest) }}'
+    secrets:
+      CR_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  release-cli-sdk:
+    needs: release-charts
```

Comment:
Does this mean the releases are always tied together? I can see us potentially iterating rapidly on the TypeScript SDK to add additional methods (it's pretty incomplete right now). But each time we implement a method, we'll be bumping the version and releasing everything in Paladin?

Reply:
No, the TypeScript release can be executed separately. 

The dependency between the chart release and SDK release is in place for scenarios where updates have been made in the SDK that aren’t yet reflected in the chart. In these cases, it’s best to release the chart first, followed by the SDK, to ensure compatibility.

However, if you’re still developing the SDK and the required functionality is already available in the chart, you can manually release the SDK.

Reply:
Perfect. I think we'll get to a point that it makes sense to release everything together. For now, I think the SDK will be playing catch-up to expose everything that Paladin already does... and I want to avoid doing releases of the operator if nothing in Paladin has actually changed, as it could be confusing to have a bunch of releases that don't actually have any new functionality.